### ISSUE TITLE
questions for the "mask"

### ISSUE THREAD

**Issue by Andyfever123** [GaryGuTC/COMG_model#15]
questions:
iu-xray mask: almost 147 G and 30 min
mimic-cxr mask: almost 179 G and 5 h

Does it mean that the mask corresponding to IU-xray data set needs 147G memory and takes 30 minutes to generate?

**Comment by GaryGuTC** [comment]
Yes, It is correct. That is the total time used by a single 3090.

**Comment by Justarrrrr** [comment]
> questions: iu-xray mask: almost 147 G and 30 min mimic-cxr mask: almost 179 G and 5 h
> 
> Does it mean that the mask corresponding to IU-xray data set needs 147G memory and takes 30 minutes to generate?
hi, i want to  ask can you tell me where  can i get the   CXAS model
